Lib/test/test_pathlib.py

```diff
@@ -1427,6 +1427,11 @@ def _check(glob, expected):
         else:
             _check(p.glob("*/fileB"), ['dirB/fileB', 'linkB/fileB'])
 
+        if not support.can_symlink():
+            _check(p.glob("*/"), ["dirA", "dirB", "dirC", "dirE"])
```

Comment:
Add also the test for `"*" + os.sep` if `os.sep != "/"`.

Add also tests for `rglob()`.